controllers/jujumachine_controller.go

```diff
@@ -309,7 +309,7 @@ func (r *JujuMachineReconciler) reconcileApplications(ctx context.Context, machi
 				ModelUUID:       modelUUID,
 				NumUnits:        1,
 				Placement: []*instance.Placement{
-					&instance.Placement{
```

Comment:
vscode  go plugin was complaining that this was redundant. go linter doesnt seem to care, but figured it best to just nix the redundant type specifier